### DISCUSSION TITLE
*.github.io certificate expires on 3/20/2024

### DISCUSSION THREAD

**Discussion by agbaber** [community/community#113208] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

The *.github.io cert expires on 3/20/2024.  This cert expiring caused a Github Pages outage last year https://www.githubstatus.com/incidents/x7njwb481j9b

Just want to make sure this is on the radar.

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for surfacing this - our team made sure this was taken care of!

**Reply by bewuethr** [reply]
Our internal pages (visibility: private) stopped working today, with the browser erroring like this:

![image](https://github.com/community/community/assets/8521043/a6f34c74-55ec-46f3-8b1b-922358628b83)

Did maybe something slip through the cracks? 😬
